python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -318,3 +313,67 @@ def quantize(self, pl_model,  # remove the type requirement for type checking
                     quantized_onnx_model)
         else:
             raise NotImplementedError("Backend {} is not implemented.".format(backend))
+
+    @staticmethod
+    def trace(model: nn.Module, input_sample=None, accelerator=None):
+        """
+        Trace a pytorch model and convert it into an accelerated module for inference.
+        For example, this function returns a PytorchOpenVINOModel when accelerator=='opevino'.
+
+        :param model: An torch.nn.Module model, including pl.LightningModule.
+        :param input_sample: A set of inputs for trace, defaults to None if you have trace before or
+                             model is a LightningModule with any dataloader attached.
+        :param accelerator: The accelerator to use, defaults to None meaning staying in Pytorch
+                            backend. Only 'openvino' is supported for now.
+        :return: A PytorchOpenVINOModel using openvino for inference if accelerator='openvino'
+        """
+        if accelerator == 'openvino':
+            return PytorchOpenVINOModel(model, input_sample)
+
+    @staticmethod
+    def save(model, path, precision=None, accelerator=None, input_sample=None):
```

Comment:
Are we going to keep `precision` and `accelerator` here? Since `model` instance already contains these.

Reply:
accelerator may be necessary here. I included torch2openvino conversion in `trainer.save` to directly save pytorch model as openvino model in local. So users don't have to run `trainer.trace` and `trainer.save` to export pytorch model. But still we can judge the saving format from the suffix of path.

Reply:
We can discuss save and load function later and modify the saved content for these functions.

Reply:
Yeah, we may keep this API for now. I think it might be a little confusing if users can have 2 ways to do the same thing.